docs/xxjitserverlocalsynccompiles.md

```diff
@@ -30,10 +30,12 @@ When you specify this JITServer option, synchronous JIT compilations are downgra
 
         -XX:[+|-]JITServerLocalSyncCompiles
 
-| Setting                 | Effect | Default                                                                            |
+| Setting                 | Effect | Default                                                                             |
 |-------------------------|--------|:----------------------------------------------------------------------------------:|
-|`-XX:+JITServerLocalSyncCompiles`           | Enable |                                                                                    |
-|`-XX:-JITServerLocalSyncCompiles`           | Disable| :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+|`-XX:+JITServerLocalSyncCompiles`           | Enable |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>                                                          |
+|`-XX:-JITServerLocalSyncCompiles`           | Disable|                             |
+
+The option `-XX:[+|-]JITServerLocalSyncCompiles` is enabled by default in most cases. The option remains disabled when you specify `-Xjit:count=0` (https://www.eclipse.org/openj9/docs/xjit/#count) and in a few advanced use cases such as running the JVM in debug mode (as described in https://blog.openj9.org/2019/04/30/introduction-to-full-speed-debug-base-on-osr/).
```

Comment:
The link to the count topic should be a proper link rather than just URL text in brackets. Same for the blog post. We should use the title of the blog post in the link and add some text to warn users that if they follow the link, they'll be leaving the OpenJ9 docs and going to the blog. eg "as described in the [Improved JVM debug mode based on OSR](https://blog.openj9.org/2019/04/30/introduction-to-full-speed-debug-base-on-osr/) post in the Eclipse OpenJ9&trade; blog" (you'll need to use Eclipse and TM since it'll be the only instance in the topic).

Reply:
I have made the changes. Thanks!